### ISSUE TITLE
Issue about hidden characters in the last column

### ISSUE THREAD

**Issue by xie186** [jmcnamara/excel-writer-xlsx#186]
I wrote the script to convert the tab delimited files to excel files using Excel::Writer::XLSX.  
Here is the link of the file:
https://gist.github.com/xie186/8937c182105a7db6a0d19f162992950b#file-tab2xlsx_mul-pl
Here is the usage if the script:
```
perl *.pl <Input file: tab-delimited file [file1,file2...]> <SheetName[Sheet1,Sheet2...]> <output file in XLSX file>
```
But I fould that the last colomn always have some hidden characters. Here is a screenshot when I select the last column:
![image](https://cloud.githubusercontent.com/assets/20909751/25131268/174bd3e0-2413-11e7-8946-2438e72da850.png)
Here is a screenshot when I select other columns:
![image](https://cloud.githubusercontent.com/assets/20909751/25131289/29c281fe-2413-11e7-9096-6967c7840d0f.png)
When I copy and paste the last column into a text file, it looks like this:
```
"0.770572333360553
"
"0.774414628044194
"
"0.685831084086936
"
"0.686595763876907
"
"0.762397414134123
"
"0.754699430153856
"
"1
"
```
Could you help me check why this happens? Thanks,
Shaojun

**Comment by jmcnamara** [comment]
Hi,

As far as I can see that is just the standard way that Excel displays data in the default number format and column width.

For example, what happens when you copy the data and paste it into a new workbook? Does it look any different than above?

John

**Comment by xie186** [comment]
Hi John, 
I think I found the problem. I tried the script on the other file which was generated in Linux environment. There was no problem for the file generated in Linux environment. The problems I reported here are all from files generated using RStudio in Windows. So I tried 'dos2unix', then the problems are gone. I think there were hidden characters in the files generated using RStudio in Windows. 
Thank you,
Shaojun
